index.js

```diff
@@ -89,7 +93,8 @@ function run() {
     require: opts.require,
     completion: opts.completion,
   }, function(env) {
-    var cfgLoadOrder = ['home', 'cwd'];
+
+    var cfgLoadOrder = ['home', 'cwd', 'initCwd'];
```

Comment:
@sttk I might be wrong but shouldn't the order be `['home', 'initCwd', 'cwd']`?  I think that `initCwd` is a fallback if we can't find one inside `cwd`, right?

Reply:
@phated Perhaps you are right, but [one test case](https://github.com/gulpjs/gulp-cli/pull/154/files#diff-73ba8d51809bb3e4ea85e2ed84a9c49eR80) fails by this because of the bug that `flags.gulpfile` does not change cwd. We should fix the bug before this.